pyrate/core/geometry.py

```diff
@@ -92,9 +108,17 @@ def calc_radar_coords(ifg, params, xmin, xmax, ymin, ymax):
     return lt_az, lt_rg
 
 
-def get_sat_positions(lat, lon, epsilon, heading, azimuth):
+def get_sat_positions(lat: np.ndarray, lon: np.ndarray, epsilon: np.ndarray,
+                      heading: np.float64, roll: np.float64) -> Tuple[np.ndarray, np.ndarray]:
     """
-        Function to calculate the lon/lat position of the satellite for a given pixel location
+    Function to calculate the lon/lat position of the satellite for each pixel.
+    :param lat: Ground latitude for each pixel (decimal degrees).
+    :param lon: Ground point longitude for each pixel (decimal degrees).
+    :param epsilon:
+    :param heading: Satellite flight heading (radians).
+    :param roll: Right or left look direction w.r.t. satellite heading; +ve = right looking (radians).
```

Comment:
I don't think 'roll' is a good choice for this parameter name. The roll angle in aviation describes a motion about the longitudinal axis (i.e. the flight direction axis), which is not what we have here. What about 'look_direction' or 'look_dir'?

See also ESA Glossary here https://earth.esa.int/handbooks/asar/CNTR5-2.html on the SAR look direction:
The radar look direction defines the angle in the horizontal plane in which the radar antenna is pointing when transmitting a pulse and receiving the return signal from the ground or from an object. The look direction is an angular measurement (in degrees) and is usually made with respect to true North. In side-looking imaging radar (SLAR), the look direction is often orthogonal (normal) to the flight trajectory (azimuth) of the platform carrying the radar and is thus synonymous with the range direction. The radar look direction is an important parameter when analysing features with a preferred orientation, for example fracture patterns in rock formations, regular street patterns, or ocean waves, as these may be enhanced through choice of appropriate radar illumination direction. ( See also "Imaging Geometry" in the geometry glossary ).

Reply:
I think your suggestion of `look_dir` is much better than my `roll`!